code/obj/item/kendo.dm

```diff
@@ -240,6 +240,23 @@
 		else
 			..()
 
+	MouseDrop(atom/over_object, src_location, over_location)
+		..()
+		var/atom/movable/screen/hud/S = over_object
+		if (istype(S))
+			playsound(src.loc, "rustle", 50, 1, -5)
+			if (!usr.restrained() && !usr.stat && src.loc == usr)
+				if (S.id == "rhand")
+					if (!usr.r_hand)
+						usr.u_equip(src)
+						usr.put_in_hand_or_drop(src, 0)
+				else
+					if (S.id == "lhand")
+						if (!usr.l_hand)
+							usr.u_equip(src)
+							usr.put_in_hand_or_drop(src, 1)
+				return
```

Comment:
```suggestion
```
useless

Reply:
return removed

Reply:
you can resolve conversations with the button press btw

Reply:
Ah I didn't realise the reviewer got a ping! Had it in my mind it was for the reviewer to resolve as a "I agree these changes resolve my comments"